src/code/z_actor.c

```diff
@@ -1850,10 +1850,10 @@ s32 Actor_ProcessTalkRequest(Actor* actor, GameState* gameState) {
 
 // Actor_PickUpExchange? Seems to be called with exchangeItemId -1 if the same actor used Actor_PickUp
 // This function is also used to toggle the "Speak" action on the A button
-s32 func_800B8500(Actor* actor, PlayState* play, f32 xzRange, f32 yRange, s32 exchangeItemId) {
+s32 func_800B8500(Actor* actor, PlayState* play, f32 xzRange, f32 yRange, PlayerActionParam exchangeItemId) {
```

Comment:
Should we also rename the args/temps that are named `exchangeItemId`?

Reply:
I would say no, because it's not acting as actionParam (see the Player struct, it's a separate field that happens to use the same enum.

Could we typedef one enum as the other?

Reply:
We could make a typedef like
```c
typedef PlayerActionParam ExchangeItemID;
```
But it feels a bit overkill and it may get confusing at some times.

I think we shouldn't make this typedef right now but revisit the idea in the future.

Reply:
Maybe leave a note above the actionparam enum that it's also used for exchangeitemid (and that we might consider a typedef in future).